diplomacy/web/src/gui/pages/page.jsx

```diff
@@ -442,7 +444,11 @@ export class Page extends React.Component {
                             {errorMessage}
                         </div>
                     </div>
-                    {this.state.body || Page.defaultPage()}
+
+                    <CatchAllErrors>
```

Comment:
Added only a catch-all error on a pretty-top level page (the Page wrapper!).

We may add better strategies to this or other apps by capturing errors in a more granular fashion, and display better fallback widgets when encountering errors.